helix-term/src/commands/lsp.rs

```diff
@@ -1473,6 +1476,44 @@ pub fn select_references_to_symbol_under_cursor(cx: &mut Context) {
     );
 }
 
+pub fn highlight_symbol_under_cursor(cx: &mut Context) {
+    let (view, doc) = current!(cx.editor);
+    let language_server = language_server!(cx.editor, doc);
+    let offset_encoding = language_server.offset_encoding();
+
+    let pos = doc.position(view.id, offset_encoding);
+
+    let future = match language_server.text_document_document_highlight(doc.identifier(), pos, None)
+    {
+        Some(future) => future,
+        None => return,
+    };
```

Comment:
```suggestion
    let Some(future) =  language_server.text_document_document_highlight(doc.identifier(), pos, None) else {
        return;
    };
```

I think helix is now in 1.65+ ?